fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/filter/SelectorFilterSpec.groovy

```diff
@@ -1,3 +1,5 @@
+// Copyright 2017 Yahoo Inc.
```

Comment:
2018

Reply:
Yahoo Inc. is not valid also anymore I think